src/views/opm.tsx

```diff
@@ -328,16 +363,28 @@ const OPM: React.FC = () => {
               </CustomButton>
             </div>
           </div>
-          <LineChart
-            title="OPM"
-            className="home-opm border-0 rounded-[10px] w-full sm:w-[89vw] lg:w-full lg:ml-[0] h-[380px] lg:h-[380px] lg:mt-[3vh] top-[-5vh]"
-            options={getChartConfig()}
-            data={data}
-          />
+          <>
+            {tabValue === 0 ? (
+              <div className="home-opm border-0 rounded-[10px] w-full sm:w-[89vw] lg:w-full lg:ml-[0] h-[380px] lg:h-[380px] lg:mt-[3vh] top-[-5vh] relative ml-[5vw] sm:ml-[1vw] mr-[5vw] sm:mr-[0] md:mr-[0] sm:h-[340px] bg-[#22262C]">
```

Comment:
md:mr-[0] can be removed as sm and md values are same

Reply:
done